.dockerignore

```diff
@@ -16,6 +16,10 @@
 /README.md
 /kustomization.yaml
 
+# Files used by the Go toolchain to embed VCS info into binaries
+!/.git/index
+!/.git/objects/
```

Comment:
At this point, wouldn't we just use `!/.git/`?

Reply:
That's probably a better approach at that point, you're right.

I'm not familiar with all the files Git uses, but for context I wanted to ensure we include only what's required by `git status`, and not the configuration of remotes, etc.

I'll try to work that out with an exclusion instead of these multiple inclusions.

Reply:
Done in [b04cad3c..bd85d927](https://github.com/triggermesh/triggermesh/compare/b04cad3ccc6dc547a24e8d23117bcd39e177bbbf..bd85d9274795c00c626e6b74825ed06dafed54d5)